lib/jekyll-import/importers/rss.rb

```diff
@@ -32,7 +32,7 @@ def self.process(options)
         source = options.fetch("source")
 
         content = ""
-        open(source) { |s| content = s.read }
+        URI.parse(source).open { |s| content = s.read }
```

Comment:
Do we need to `require "uri"` for this?

Reply:
Not sure, do we need to when it's in the standard lib?
https://ruby-doc.org/stdlib-2.3.4/libdoc/uri/rdoc/URI.html

Reply:
Other files use URI.decode without requiring, so I thought it was not mandatory.